charts/airflow/templates/config/configmap-env.yaml

```diff
@@ -101,7 +101,10 @@ data:
   ## Airflow - KubernetesExecutor
   ## ----------------
   {{- if (not .Values.airflow.config.AIRFLOW__KUBERNETES__ENV_FROM_CONFIGMAP_REF) }}
-  AIRFLOW__KUBERNETES__ENV_FROM_CONFIGMAP_REF: "{{ include "airflow.fullname" . }}-env"
+  AIRFLOW__KUBERNETES__ENV_FROM_CONFIGMAP_REF: "{{ include "airflow.fullname" . }}-env{{include "airflow.kubernetes.configMaps" .}}"  
```

Comment:
Part of me thinks it would be more clear if the comma was explicitly put here, rather that it appearing from `{{include "airflow.kubernetes.configMaps" .}}`

nit: please put spaces between `{{`

For example: 
```
AIRFLOW__KUBERNETES__ENV_FROM_CONFIGMAP_REF: {{ include "airflow.fullname" . }}-env,{{ include "airflow.kubernetes.configMaps" . }}
```

Reply:
@thesuperzapper  I agree it may be more clear, but if they do not include extra config maps there will be a trailing comma which may be confusing when looking at the environment variable. I can update it to do that if that is what you would like.